apps/mail/app/(routes)/mail/compose/handle-mailto/route.ts

```
@@ -78,49 +95,140 @@ async function parseMailtoUrl(mailtoUrl: string) {
+            // For the next attempt, try a different format for the To field
+            if (Array.isArray(draftData.to)) {
+              // Convert array to comma-separated string
+              draftData.to = draftData.to.join(',');
+            } else if (typeof draftData.to === 'string') {
+              // Convert string to array
+              draftData.to = draftData.to.split(',').map((e: string) => e.trim());
+            }
+          }
+          
+          await delay(RETRY_DELAY * attempt); // Exponential backoff
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)